src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -11143,6 +11143,22 @@ instruct mulHiL_rReg(iRegLNoSp dst, iRegL src1, iRegL src2, rFlagsReg cr)
   ins_pipe(lmul_reg_reg);
 %}
 
+instruct umulHiL_rReg(iRegLNoSp dst, iRegL src1, iRegL src2, rFlagsReg cr)
+%{
+  match(Set dst (UMulHiL src1 src2));
+
+  ins_cost(INSN_COST * 7);
+  format %{ "umulh   $dst, $src1, $src2, \t# umulhi" %}
```

Comment:
There's an unnecessary trailing comma here and in the `smulh` pattern above.

Reply:
Hi, I tried the JMH benchmark with this patch, it does generate the "umulh" instruction instead of smulh.

Reply:
Thanks @Bhavana-Kilambi, as the `hsdis` output above shows, this is indeed the case.